File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/network/ApiInterface.java
package com.tiff.tiffinbox.Seller.network;

/**
 * Created by abhisheksisodia on 2017-10-06.
 */

//import com.abhi.inbox.model.Message;
import com.tiff.tiffinbox.Seller.Model.Message;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiInterface {

    @GET("inbox.json")
    Call<List<Message>> getInbox();

}

File: TiffinBox/app/src/main/java/com/tiff/tiffinbox/Seller/network/ApiInterface.kt
package com.tiff.tiffinbox.Seller.network

import com.tiff.tiffinbox.Seller.Model.Message
import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by abhisheksisodia on 2017-10-06.
 */
//import com.abhi.inbox.model.Message;
interface ApiInterface {
    @get:GET("inbox.json")
    val inbox: Call<List<Message?>?>?
}